docs/src/components/NavLink.js

```diff
@@ -19,10 +19,10 @@ const NavLink = ({ children, to, history }: Props) => {
   const href = history.createHref({
     pathname: to,
   });
-  const { isSidebarOpen, setIsSidebarOpen } = useSidebarContext();
+  const { isSidebarOpen, setIsSidebarOpen } = useNavigationSidebarContext();
 
   const handleClick = ({ event }) => {
-    setIsSidebarOpen(!isSidebarOpen);
+    setIsSidebarOpen(isSidebarOpen && !isSidebarOpen);
```

Comment:
@AlbertCarreras Should this just be `false`?

Reply:
@christianvuerings  This was a fix! We only want to close it if it was opened with the menu button. Otherwise every time you click an option it changes. Sometimes I would find the sidebar open when reducing the screen, and it was very annoying. You want to click the hen by button first, then make it disappear after selecting an option

Reply:
set to 'false'